packages/core/src/components/advanced-select/advanced-select.scss

```diff
@@ -0,0 +1,186 @@
+@use "sass:math";
+@use "~dso-toolkit/src/utilities";
+@use "~dso-toolkit/src/global/mixins/set-colors.mixin" as set-colors;
+@use "~dso-toolkit/src/components/advanced-select";
+
+:host {
+  display: block;
+}
+
+@include utilities.box-sizing();
+
+.active-option {
+  display: flex;
+  width: 100%;
+  justify-content: space-between;
+  border: 1px solid advanced-select.$border-color;
+  border-radius: advanced-select.$border-radius;
+  cursor: pointer;
+  background-color: advanced-select.$bg-color;
+  line-height: inherit;
+  font-family: inherit;
+  font-size: inherit;
+
+  &.open {
+    border-radius: advanced-select.$border-radius advanced-select.$border-radius 0 0;
+  }
+
+  &-label {
```

Comment:
Doe dit liever niet. als je aan het debuggen bent, en je zoekt in de codebase naar `.active-option-label` ga je hem nooit vinden.

Reply:
Heb het aangepast. Ik vond dit juist een fijne manier van schrijven in SCSS maar als dit niet gebruikelijk is binnen het project dan doe ik het liever zoals jij het voorstelt.